pkg/providers/amifamily/resolver.go

```diff
@@ -291,6 +292,7 @@ func (r DefaultResolver) resolveLaunchTemplates(
 			CapacityType:            capacityType,
 			CapacityReservationID:   id,
 			CapacityReservationType: capacityReservationType,
+			Tenancy:                 *nodeClass.Spec.Tenancy,
```

Comment:
```suggestion
			Tenancy:                 aws.ToString(nodeClass.Spec.Tenancy),
```

Reply:
What happens if there is not a dedicated host available, is it more desirable to fallback to Default instances for an instance type or to keep trying different instances as defined by the NodePool?

Reply:
I also would add that ultimately this is just configuring a vendor provided backend.  I'm not sure how we'd even begin to do any kind of fall back intelligently wrt Tenancy given the LaunchTemplate api.